k8s/Dockerfile

```diff
@@ -46,7 +49,7 @@ RUN groupadd -r cassandra --gid=999 && useradd -r -g cassandra --uid=999 --creat
 COPY --from=base /install /usr/local
 
 # wget could happen in the build-phase
-RUN apt-get update && apt-get install -y python3 wget
+RUN apt-get update && apt-get install -y python3 python3-setuptools wget
```

Comment:
You could add `&& rm -rf /var/lib/apt/lists/*` to the end of this line to clean up the Apt cache, but it only saves 30-40 MB of space. I know you are looking to reduce the image size more than that, but every little bit helps.

Reply:
You are right I was able to save about 30 MB. Thanks for the tip!